src/core/Map.h

```diff
@@ -10,17 +10,20 @@
 #include <vector>
 
 enum Owner {
-    NEUTRAL = 0,
-    CONTESTED = 1,
-    ENEMY = 2,
-    SELF = 3,
+    NEUTRAL = 0,  // No Townhalls at expansion, no workers enroute
+    CONTESTED = 1,  // Self Worker enroute to build TownHall at expansion
```

Comment:
Originally, CONTESTED meant to be 'we and a enemy has units in the same region', i.e. this region should be target for attach or defence (if we have any structures there). This is not related to issued orders.
If you want to use this enum, perhaps it would be better to add a new value e.g. 'CAPTURING'.

Reply:
Ok, added to description. This can be updated in 'Secure/Defend new expansion' issue (#53) to make use of CONTESTED as intended.